app/assets/main/react_components/footprint_form/FootprintForm.jsx

```diff
@@ -10,13 +11,31 @@ import OptionNumerical from './OptionNumerical.jsx';
  */
 const FootprintForm = ({ calculator, questions, options, footprint }) => {
 
-  const order = ["region", "home", "home_area", "heating", "green_electricity", "food", "shopping", "car_type", "car_distance", "flight_hours"]
+  const [questionCategories] = useState({"region": "home", "home": "home", "home_area": "home", "heating": "home", "green_electricity": "home", "food": "utensils", "shopping": "shopping-bag", "car_type": "car", "car_distance": "car", "flight_hours": "plane"});
```

Comment:
Detta behöver inte vara ett state uppfattar jag? I och med att sen setter aldrig används.

```suggestion
  const questionCategories = {"region": "home", "home": "home", "home_area": "home", "heating": "home", "green_electricity": "home", "food": "utensils", "shopping": "shopping-bag", "car_type": "car", "car_distance": "car", "flight_hours": "plane"};
```